wAbout/Logging-In.md

```diff
@@ -81,9 +81,9 @@ To switch between accounts, simply click the link of the account you wish to swi
 
 ### Forgetting Login Credentials
 
-Forgetting credentials means the login entry is deleted from the cache and removed from the Login Manger. You will have to reenter your credentials the next time you wish to log in.
+Forgetting credentials means the login token is deleted from the cache and the profile manager removed from the Login Manger. You will have to reenter your credentials the next time you wish to log in.
```

Comment:
it kind of seems weird to have profile manager in lowercase and login manager in uppercase

Reply:
Yea I need help standardizing these caps :/